docs/internal_api.yml

```diff
@@ -5,7 +5,7 @@ info:
   description: |
     Internal API of deployments service
 
-host: 'docker.mender.io'
```

Comment:
This API is not exposed on hosted mender, and it is considered internal.
I don'w know what the policy is for these end-points, but hosted.mender.io is wrong.